src/post.rs

```diff
@@ -107,6 +107,12 @@ fn parse_comments(json: &serde_json::Value, post_link: &str, post_author: &str,
 
 			let score = data["score"].as_i64().unwrap_or(0);
 
+			// The API only provides comments up to some threshold.
+			// Further comments have to be loaded by subsequent requests.
+			// The "kind" value will be "more" and the "count"
+			// shows how many more comments exist in the current nesting level.
+			let more_count = data["count"].as_i64().unwrap_or_default().to_string();
```

Comment:
Do we need this to be a string value? I'm pretty sure we can format as an i64 directly into askama. Not that it's a huge or expensive allocation, but if the added layer of misdirection is unnecessary, it's always a good idea to reduce it!

Reply:
Adjusted it, thanks.